src/main/java/com/linecorp/sample/login/infra/line/api/v2/LineAPIService.java

```diff
@@ -63,33 +63,39 @@ public AccessToken refreshToken(final AccessToken accessToken) {
     }
 
     public Verify verify(final AccessToken accessToken) {
-        return getClient(t -> t.verify(accessToken.access_token));
+        return getClient(t -> t.verify(
+                accessToken.access_token,
+                channelId,
+                channelSecret));
     }
 
     public void revoke(final AccessToken accessToken) {
-        getClient(t -> t.revoke(accessToken.refresh_token));
+        getClient(t -> t.revoke(
+                accessToken.refresh_token,
+                channelId,
+                channelSecret));
     }
-
+    
     public Profile profile(final AccessToken accessToken) {
         return getClient(t -> t.profile(addBearer(accessToken)));
     }
 
-
-    public String getLineWebLoginUrl(String state) {
+    public String getLineWebLoginUrl(String state, String nonce) {
         final String encodedCallbackUrl;
         try {
             encodedCallbackUrl = URLEncoder.encode(callbackUrl, "UTF-8");
         } catch (UnsupportedEncodingException e) {
             throw new RuntimeException(e);
         }
 
-        return "https://access.line.me/dialog/oauth/weblogin?response_type=code"
+        return "https://access.line.me/oauth2/v2.1/authorize?response_type=code"
                 + "&client_id=" + channelId
                 + "&redirect_uri=" + encodedCallbackUrl
-                + "&state=" + state;
+                + "&state=" + state
+                + "&scope=openid%20profile"
```

Comment:
@KosukeKamiya 
scope parameter value is only "openid%20profile" ?
Does scope parameter have other value?